### DISCUSSION TITLE
编译一直卡在这里，怎么办

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#141] [Q&A]
![image](https://user-images.githubusercontent.com/45897471/169327374-0364aeca-bdc6-4d4f-b285-a7b9232366a8.png)
你好，编译一直卡在这里不动，这该怎么办。
谢谢

**Comment by earthmoon** [comment]
![image](https://user-images.githubusercontent.com/45897471/169332742-6ed06472-5521-417b-9a40-75d67488b35d.png)
Vscode显示这种错误。。。

**Comment by tanukihee** [comment]
超容量了？你的mhchem写了啥

**Reply by earthmoon** [reply]
`\usepackage[version=4]{mhchem} %化学包`
这个应该是对的吧

**Reply by earthmoon** [reply]
不知道重试了多少次，图片上的这两个轮流来。现在好了，感觉不稳定啊。
![image](https://user-images.githubusercontent.com/45897471/169335825-b7b6ee7e-ef09-4d41-9aa9-c98700bd41f1.png)

**Reply by earthmoon** [reply]
![image](https://user-images.githubusercontent.com/45897471/169336048-9ac302bd-a13c-40fc-8ea4-392355aca3cf.png)
要么就是这个代码有问题，反正是动了这个代码

**Reply by tanukihee** [reply]
> `\usepackage[version=4]{mhchem} %化学包`
> 这个应该是对的吧

我只关心你用化学包写了什么

**Reply by earthmoon** [reply]
只写了化学式和化学方程式。

> \let\oldnumberline\numberline%
        \renewcommand{\numberline}{\figurename~\oldnumberline}%

这两行删去就正常了，问题出在哪？很奇怪。

**Reply by earthmoon** [reply]
找到原因了，必须这样才行：

> {%
        \let\oldnumberline\numberline%
        \renewcommand{\numberline}{\figurename~\oldnumberline}%
        \listoffigures%
        }
        \__whu_new_chapter_page:

`{`和`}`不能省略
